### PR TITLE
Fixes video-game-coding-club/crypto-webpage#79

### PR DESCRIPTION
**Pull request by zmedin**
Signed-off-by: zmedin <zmedin@gmail.com>

### CODE REVIEW

File: helpers.js [hunk 1]
```diff
@@ -28,10 +28,42 @@ window.processForm = processForm;
...
+  var Zcode = 'Z'.charCodeAt(0);
+  var acode = 'a'.charCodeAt(0);
+  var zcode = 'z'.charCodeAt(0);
+
+  // Only shift if the letter really is a letter.
+  if (code >= Acode && code <= Zcode) {
+    // The letter is a capital letter. Shift it.
+    code += shift;
+    // If code is too big or too small, wrap it so it isn't.
+    code = wrapNumber(code, Acode, Zcode);
```
Comment (nicolasbock) [inline helpers.js]: You could simplify this and the conditionals with using the modulo operator:

```suggestion
    code = 'a'.charCodeAt(0) + letter.charCodeAt(0) % 26
```

**Review by nicolasbock — CHANGES_REQUESTED** [review]
That's a great start! Could you also have a look at the tests in https://github.com/video-game-coding-club/crypto-webpage/blob/main/__tests__/helpers-test.js#L11 and add a test that tests the wrapping, i.e. `z + 1 -> a`.

File: helpers.js (hunk 1 above)
Comment (zmedin) [reply helpers.js]: Dad's comment: Javascript % doesn't work as expected for negative shifts, for example -2 % 26 = -2, not 24. So this way still requires additional logic. But we will think about this some more.

Comment (nicolasbock) [reply helpers.js]: Hmm, yes, that's inconvenient :/

Do you just want to add a unit test for now and leave the shift function as it is?

Thanks!

Comment (zmedin) [reply helpers.js]: I think it works now.

Comment (nicolasbock) [reply helpers.js]: Nice! :+1: 

Do you need help with the unit test?

Comment (zmedin) [reply helpers.js]: No, thank you.

Comment (zmedin) [reply helpers.js]: All done.

**Review by nicolasbock — APPROVED** [review]
Thank you!

**Pull request merged**